extensions/wikia/PortableInfobox/controllers/ApiPortableInfobox.class.php

```diff
@@ -107,11 +108,17 @@ public function getVersion() {
 	}
 
 	/**
-	 * @return mixed
+	 * @return array of key-value args
```

Comment:
PHPDOC format is ```@return datatype description``` so description shoudn't be ```"of key-value..."``` but just ```"key-value..."```
Also - seems like it got split into two lines too early